ginga/rv/plugins/Cuts.py

```diff
@@ -444,6 +456,33 @@ def set_cutsdrawtype_cb(self, w, index):
         self.cuttype = self.cuttypes[index]
         self.canvas.set_drawtype(self.cuttype, color='cyan', linestyle='dash')
 
+    def copy_cut_cb(self, w):
+        old_tag = self.cutstag
+        if old_tag == self._new_cut:  # Can only copy existing cut
+            return
+
+        from copy import deepcopy
+
+        new_index = self._get_new_count()
+        new_tag = "cuts{}".format(new_index)
+
+        old_obj = self.canvas.get_object_by_tag(old_tag)
+        self.logger.debug("copying {}".format(old_obj.objects[0]))
+
+        # FIXME: deepcopy breaks because logger cannot be pickled
+        new_cut = self._create_cut_obj(
+            new_index, deepcopy(old_obj.objects[0]), color='cyan')
```

Comment:
@ejeschke , I can't get `deepcopy` to work. And the existing API does not appear to have a `copy` method. Any advise? Thanks!

Reply:
@pllim, recommend adding a `copy()` method to `~ginga.canvas.CanvasObject`, so that we can benefit from it in the future (then just call `copy()` on the object here).

For implementing `copy()` I think you can so something like:
```python
from copy import deepcopy
...
def copy(self): 
    logger = self.logger
    try:
        self.logger = None
        obj = deepcopy(self)
        obj.logger = logger
        return obj
    finally:
        self.logger = logger
```
There might be some other things that should not be shared--we can figure those out later.

Reply:
See code comment under "Files changed"

Reply:
Hmm, just tried this and now it fails trying to deepcopy `threading.RLock` objects.